evaluation/metrics/train_file_metrics.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from sklearn.multioutput import MultiOutputClassifier
+from sklearn.svm import LinearSVC
+
+
+def dataperf(train, test, constructor_args):
+    from .dataperf_embeddings import dataperf_embeddings
```

Comment:
Are these the final values for embeddings (from douwe's previous comments, I thought these were just random embeddings so that we could demo how the flow looks like)?

Reply:
(also is this function used anywhere else? I tried to search for something that calls this but could not)

Reply:
Yes these are random embeddings for test purposes. I need to get the final values from the dataperf team before this code will actually work with their datasets.